book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] It would be better if we set the getter and setters for our rentals since we are not only reading it but also writing it. Kindly check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) for more information. For example, a book has many rentals kind of association

Reply:
applied suggested changes 👍🏽